packages/search-field/src/components/Menu.js

```diff
@@ -49,18 +50,20 @@ const Menu = ({
     return null;
   }
   return (
-    <DropdownMenuWrapper onMouseDown={onMenuMouseDown}>
-      <ScrollBox
-        maxHeight={maxMenuHeight}
-        onReachEnd={onScrollReachEnd}
-        ref={innerRef}
-      >
-        {items.map((item, index) =>
-          renderItem(item, onItemClick, index === highlightedIndex)
-        )}
-        {footer}
-      </ScrollBox>
-    </DropdownMenuWrapper>
+    <MenuWrapper>
+      <DropdownMenuWrapper onMouseDown={onMenuMouseDown}>
```

Comment:
Since `DropdownMenuWrapper` is a styled component, we can extend it and set new styles to it.

It'd be like this

```
import { DropdownMenuWrapper } from "@crave/farmblocks-dropdown";

const ExtendedWrapper = styled(DropdownMenuWrapper)`
  top: 72px;
  position: absolute;
  width: 100%;
`;
```

For reference: https://github.com/CraveFood/farmblocks/blob/c516cf3380175d5308ace6a8b54f1508446ba3b7/packages/input-select/src/styledComponents/DropdownWrapper.js